cockatrice/src/localserver.cpp

```diff
@@ -11,7 +11,7 @@ LocalServer::LocalServer(QObject *parent)
 
 LocalServer::~LocalServer()
 {
-    prepareDestroy();
+    prepareDestroy(false);
```

Comment:
What about making `prepareDestroy` virtual and overriding the behavior differently in real server and local server mode?